src/routes/Home.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Searchbar from '../components/Searchbar';
+import Logo from '../components/Logo';
+import Picture from '../components/Picture';
+
+const Home = () => {
+  const { filt } = useSelector((state) => state.apod);
+  return (
+    <div className="h-screen font-lato bg-blue-500">
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure your logo `NASA` and the images on your home page have a good feel and touch, as you may notice how they are messed up.
`Hint: You may add some padding around your home logo and reduce the size of your image and make them responsive so that they may look better.[some width in % and a height=auto may work fine]`
![image](https://user-images.githubusercontent.com/97100091/218087419-97349965-7ac8-4192-9c18-294555f0fdb3.png)